source/effect_preprocessor.hpp

```diff
@@ -144,8 +151,8 @@ namespace reshadefx
 		size_t _current_input_index = 0;
 		unsigned short _recursion_count = 0;
 		location _output_location;
-		std::unordered_set<std::string> _used_macros;
-		std::unordered_map<std::string, macro> _macros;
+		std::map<std::string, location> _used_macros;
+		std::map<std::string, macro> _macros;
```

Comment:
Is there a reason these were turned into a `std::map`, rather than `std::unordered_map`?

Reply:
I prefer to sort the keys (definition names) in definition editor.
But well, following code might be better.
```cpp
// Keep track of used preprocessor definitions (so they can be displayed in the GUI)
effect.definitions.clear();

for (const auto &definition : pp.used_macro_definitions())
{
	if (definition.first.size() <= 4 || ... )
		continue;

	effect.definitions.push_back( ... );
}

std::sort(effect.definitions.begin(), ... ); // ** Add **

// Keep track of included files
effect.included_files = pp.included_files();
```